paging-runtime-composeui/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.jetbrains.compose.compose
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinMultiplatform
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+
+plugins {
+  alias(libs.plugins.kotlin.multiplatform)
+  alias(libs.plugins.mavenPublish)
+  id("org.jetbrains.compose")
+  id("com.android.library")
+  id("kotlin-parcelize")
```

Comment:
Couldn't get this to work with type-safe reference through catalog... should look into this again